github-orgs/kubeflow/org.yaml

```diff
@@ -1213,12 +1190,13 @@ orgs:
           wg-pipeline-leads:
             description: Team of Pipeline Working Group leads
             members:
-            - animeshsingh
             - chensun
+            - droctothorpe
             - james-jwu
-            - IronPan
-            - zijianjoy
-            privacy: closed
+            - HumairAK
+            - mprahl
+            - zazulam
+            privacy: open
```

Comment:
Thanks for the update @anishasthana!
You might also want to give write permission to `kubeflow/pipelines` repo for @kubeflow/wg-pipeline-leads team